src/specifications/profile-resolution/profile-resolution-specml.xml

```diff
@@ -3,1772 +3,1370 @@
-              XML and JSON requirements are different.</revisit></p>
-          <p>When a catalog is organized by a <src>merge/custom</src> directive, loose parameters
-            are copied into the resulting grouping structure directly preceding the first control
-            appearing that contains an <src>insert</src> directive referencing that parameter.</p>
-          <p>Parameters that are not referenced by an <src>insert</src> elsewhere in the target, do not appear
-            in the target. While it is not formally invalid to include unreferenced (unused) parameters,
-          profile resolution requires that none be included except as a special option.</p>
+        <section id="import-param">
+          <head>Handling Params</head>
+          <p>Any <src>param</src> that is not directly under a control is referred to as a
+              <q>loose</q> param. </p>
+          <p>All loose params from both imported documents and the profile source MUST be included
+            implicitly. Enterprising implementations may consider pruning unused params at this
+            point <xref rid="cleanup"/>, but care should be taken to not prune future needed
+            params.</p>
         </section>
       </section>
+      <section id="mapping">
+        <!-- TODO: Remove generic definition, Add below types:
+      mapping:
+        control
+        group
+        param
+        role 
+        part 
+       --> 
+        
+        <head>Mapping Controls</head>
+        <p>The optional "mapping" child of a given "import" provides a simple ID remapping for
+          objects included from that specific import. This provides the means for profile authors to
+          proactively avoid clashing IDs of controls and other objects. </p>
+        <p>The Mapping section consists of 5 optional subsections, each covering a particular type
+          of object. Each subsection is a list of ID mappings to be applied for objects that are the
+          parent object type.</p>
+        <p>When encountering a given mapping instruction, processors:</p>
+        <ul>
+          <li>
+            <p>MUST locate the included object in the intermediate whose distinctive ID matches the
+              value of the "from" object and whose object type is the same as the name as the
+              parent. If none is found, the processor SHOULD provide a warning, but MAY continue
+              processing. In the case there are more than one included objects of the specified type
+              with the same distinctive ID, behavior is unspecified, as this means the source
+              catalog was malformed.</p>
+          </li>
+          <li>
+            <p>MUST change the distinctive ID of that object to be equal to the value of the "to"
+              object.</p>
+          </li>
+          <li>
+            <p>SHOULD from this point forward only consider (or even be aware of) the new value of
+              that object's distinctive ID.</p>
```

Comment:
Maybe phrase this as "all references to the old ID MUST be updated, allowing the new ID to be used in subsequent profile sections." or something similar.